mmcv/runner/hooks/logger/wandb.py

```diff
@@ -8,13 +8,14 @@
 class WandbLoggerHook(LoggerHook):
 
     def __init__(self,
-                 log_dir=None,
+                 init_kwargs=dict(),
```

Comment:
Use `None` as the default value to avoid potential issues.

Reply:
The default key of this key should be an empty dict(), which can init the Writer if there are no init_kwargs given.